packages/desktop-client/src/components/accounts/Reconcile.tsx

```diff
@@ -164,6 +167,11 @@ export function ReconcileMenu({
           />
         </InitialFocus>
       )}
+      <Text style={{ color: theme.pageTextSubdued, paddingBottom: 4 }}>
+        {account?.last_reconciled
+          ? `${t('Reconciled')} ${tsToRelativeTime(account.last_reconciled, locale)}`
```

Comment:
Instead of directly using string templating, it would be good to use translation placeholders, so that languages with a different word order may still work. Something like this should work:

```suggestion
          ? t('Reconciled {{ relativeTimeAgo }}', { relativeTimeAgo: tsToRelativeTime(account.last_reconciled, locale) })
```